### PR TITLE
Footer added

### PR DESCRIPTION
**Pull request by aditimaurya**
Hey @grapheo12, I've added the banner of metakgp along with the project link and metakgp's home page link in the footer. As the colour of the footer wasn't mentioned so I've taken it as white by default and the colour of the text is black. Please mention if you need some specific colours in them. ( files I've altered are main.css and home.html)
Do let me know whether the edits are up to mark or not.

### CODE REVIEW

**Comment by grapheo12** [comment]
@aditimaurya I hope links are working now.
Few more things:
1. **Never send PR from your master branch.**  Always create a new branch in your repo and work from that. It will save a lot of merge conflicts in big projects. It's fine this time. Keep this in mind from the next time.
2. **Squash your commits.** You have 2 related commits, so it is better to merge them together. It will help keep the Git history clean. Also "altered" is not a very good commit message. While squashing provide a good commit message briefly describing what you did. So that, 5 years later when you will go through the commit history, you will get to know what you did at a glance.

Let us know if you are facing issues with Squashing. You don't need to create a new PR. Just force push to your master.

Lastly, please attach a screenshot. It will be helpful in reviewing. (I am too lazy to test it out myself, you know).
Provide a screenshot in both Desktop and Mobile mode.

**Review by grapheo12 — APPROVED** [review]
Code wise LGTM.
@thealphadollar can merge after she squashes.

File: templates/home.html [hunk 1]
```diff
@@ -142,7 +142,9 @@ <h4 class="unlockFont">Upload ICS file of your timetable from <a href="https://g
 					<input id="file-in" type="file" class="form-control col-auto" onchange="readICS()">
 				</div>
 				<div id="details-div"></div>
-
+				<p id=".footer">
+				<strong>Contribute to this project on <a href="https://github.com/metakgp/what-slot"> GitHub </a> | powered by <a href="https://metakgp.github.io/">metakgp</a> with ♥ </strong>
```
Comment (thealphadollar) [inline templates/home.html]: Yo Aditi, do you think having P and M capital in "powered" and "metakgp" respectively will look better?

**Review by thealphadollar — COMMENTED** [review]
Hey @aditimaurya Can you please add screenshots - one for desktop and another for mobile for the change in the PR description? 

You can use [this reference](https://www.browserstack.com/guide/view-mobile-version-of-website-on-chrome) to know how to see mobile view in chrome itself.

File: templates/home.html [hunk 2]
```diff
@@ -142,7 +142,9 @@ <h4 class="unlockFont">Upload ICS file of your timetable from <a href="https://g
 					<input id="file-in" type="file" class="form-control col-auto" onchange="readICS()">
 				</div>
 				<div id="details-div"></div>
-
+				<p id=".footer">
+				<strong>Contribute to this project on <a href="https://github.com/metakgp/what-slot"> GitHub </a> | Powered by <a href="https://metakgp.github.io/">Metakgp</a> with ♥ </strong>
```
Comment (thealphadollar) [inline templates/home.html]: Please remove the spaces around "Github". I should have included it in the first comments itself, thank you for persisting and your patient :smile: 

The space after heart could be removed.

File: templates/home.html [hunk 3]
```diff
@@ -142,7 +142,9 @@ <h4 class="unlockFont">Upload ICS file of your timetable from <a href="https://g
 					<input id="file-in" type="file" class="form-control col-auto" onchange="readICS()">
 				</div>
 				<div id="details-div"></div>
-
+				<p id=".footer">
```
Comment (thealphadollar) [inline templates/home.html]: Are you sure there should be ".footer" as the ID? IIRC, "." before ID is required only in the CSS and here you can directly add `id="footer"`.

**Comment by grapheo12** [comment]
Hey @aditimaurya Why did you close this?

**Comment by aditimaurya** [comment]
@grapheo12, I accidentally close it. I faced some problems while merging the commits, and it all happened. I'll be making a new pull request along with the screenshots of the web and mobile view.

**Pull request closed without merging**